dashboard/app/models/user.rb

```diff
@@ -2570,6 +2571,11 @@ def dependent_students
     dependent_students
   end
 
+  def dependent_students_count
```

Comment:
Consider adding this to `queries/user.rb`. That file doesn't exist, but it would be easy to setup.

Reply:
Good call, I agree.